### DISCUSSION TITLE
Not able to set the Gain  - RPi4B 8GB with Rpi HQ Camera

### DISCUSSION THREAD

**Discussion by baryah** [aaronwmorris/indi-allsky#607] [Q&A] [answered]
I have set the night time gain to 5 and moon mode gain to 2.

but the pictures in moon mode are being captured with gain 16, which is the maximum that RPi HQ camera allows

**Comment by aaronwmorris** [comment, marked as answer]
If you changed the gain during operation, you may have to fully restart the indi-allsky service to use the new settings.  There are a few cases where a RELOAD is not sufficient.
